### ISSUE TITLE
Error running 'ideviceinstaller -i /Users/morhaham/Desktop/cordova/myApp/platforms/ios/build/device/HelloCordova.ipa'

### ISSUE THREAD

**Issue by morhaham** [Microsoft/vscode-cordova#272]
I've tried to reinstall everything, delete the project and create a new one.
Also, I can run the application from the terminal with cordova run ios --device and from Xcode.

 - InspectingPackage (20%)

 - TakingInstallLock (20%)

 - PreflightingApplication (30%)

 - InstallingEmbeddedProfile (30%)

 - VerifyingApplication (40%)

 - CreatingContainer (50%)

 - InstallingApplication (60%)

 - PostflightingApplication (70%)

 - SandboxingApplication (80%)

 - GeneratingApplicationMap (90%)
Error running 'ideviceinstaller -i /Users/morhaham/Desktop/cordova/myApp/platforms/ios/build/device/HelloCordova.ipa'

**Comment by aocarmo** [comment]
Any solution? i have the same issue! =/

**Comment by jalchr** [comment]
Me too ....

**Comment by kirajhpang** [comment]
same situation after I update package from brew
```
brew update && brew upgrade libimobiledevice --HEAD && brew upgrade ideviceinstaller ios-webkit-debug-proxy
```

**Comment by harleybl** [comment]
I am also in the same situation. 
I upgraded to 1.2.10 of Cordova Tools.
Used brew to update libmobiledevice, ideviceinstaller and ios-webkit-debug-proxy

Any ideas?

**Comment by kirajhpang** [comment]
@harleybl I think I somehow solve it accidentally. From what I still remember, I think I recreate the entire platform. and make sure `ios-sim` is install, for Cordova to understand what simulator u have. 

this is step normally I do for ionic Cordova project

`Cordova platform add ios`
`Cordova prepare`

then make u install `ios-sim`
`npm install ios-sim@latest --save` <== yes, install in project level
then follow some doc, they said you should also install in global level
`npm install -g ios-sim@latest --save`

because you just create platform, you need open Xcode with that project u create after `Cordova platform add ios` then sign the project and you can start use latest vscode-cordova to debug. 

**Correct me if I am wrong, hope this help somewhere**

**Comment by iTOYS** [comment]
Fixed #322
